internal/highway/crud_registry.go

```diff
@@ -1,57 +1,122 @@
 package highway
 
 import (
-	context "context"
-	"fmt"
+	"context"
 	"net/http"
 
 	"github.com/gin-gonic/gin"
 	rtv1 "github.com/sonr-io/sonr/x/registry/types"
 	rt "go.buf.build/grpc/go/sonr-io/blockchain/registry"
 )
 
+// ParamsHTTP
+// @Summary
+// @Schemes
+// @Description WhoIsHTTP
+// @Tags Registry
+// @Produce json
+// @Success      200  {string}  message
+// @Failure      500  {string}  message
+// @Router /registry/who_is[GET]
+func (s *HighwayServer) ParamsHTTP(c *gin.Context) {
+	params, err := s.cosmos.QueryParams()
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
+		return
+	}
+
+	c.JSON(http.StatusOK, params)
+}
```

Comment:
I dont think we need to expose query params over http.

Reply:
Can I follow https://www.notion.so/sonr/ADR-1-Sonr-DID-Method-Specification-0677c6f389794787bf40772bb00f6f8f#ae710980849747ecb699dc83265436b4 as reference?

Reply:
Let me follow up on this but for now ill resolve the comment. thanks!

Reply:
Done